targets/targets.json

```diff
@@ -1200,6 +1200,11 @@
     },
     "MTB_ADV_WISE_1530": {
         "inherits": ["USI_WM_BN_BM_22"],
+        "config": {
+            "led1": "PA_4",
+            "led2": "PC_12",
+            "led3": "NC"
```

Comment:
same comment as above. why is led3 required if not connected? Also, this may not be for this particular PR, but looks like there is a naming issue here.. the parent target is "USI_xxx" (note: manf = USI), and ADV target inherits from it.. (note: manf = ADV). We may want to change this in the future.

Reply:
MTB has 3 leds, so all MTB targets should define those. But unfortunately all MCBs don't connect those leds, so those that don't need `NC` instead.